examples/lead-generation/airtop_client.py

```diff
@@ -0,0 +1,56 @@
+from airtop import Airtop, SessionConfigV1, SessionResponse, WindowResponse
+from env_config import env_config
+
+default_session_config: SessionConfigV1 = SessionConfigV1(
+    persist_profile=False,
+    timeout_minutes=10
+)
+
+class AirtopClient:
```

Comment:
Is there a reason for creating your own AirtopClient wrapper with your own methods for creating sessions and windows? It feels a little odd in a recipe showing off our SDK, like maybe our SDK is missing some pieces. If it is, that's probably worth discussing.